### PR TITLE
Update run_tests_pipeline.py to allow us to rerun failed suites and eliminate itermittent failures

### PR DESCRIPTION
**Pull request by duncan-at-hiveit**

### CODE REVIEW

**Review by markhiveit — APPROVED** [review]

File: tests/robot-tests/scripts/run_tests_pipeline.py [hunk 1]
```diff
@@ -54,5 +57,7 @@ def run_tests_pipeline():
 
     parser.add_argument("--processes", dest="processes", help="number of processes to run", required=True)
 
+    parser.add_argument("--rerun-failed-suites", dest="rerun_failed_suites", help="rerun any failed suites from a previous run", required=False)
```
Comment (benoutram) [inline tests/robot-tests/scripts/run_tests_pipeline.py:60]: Does this need a UI test pipeline change to add the extra argument, across all of the UI test tasks in the build pipeline and release pipeline

Comment (duncan-at-hiveit) [reply tests/robot-tests/scripts/run_tests_pipeline.py:60]: Done already ta

Comment (duncan-at-hiveit) [reply tests/robot-tests/scripts/run_tests_pipeline.py:60]: Ooh but not in the Release pipeline though... I'll see if it does the trick in the build pipeline first and then apply to the release one if it's all good if that's OK?

Comment (benoutram) [reply tests/robot-tests/scripts/run_tests_pipeline.py:60]: Yep that's cool. 🙂 

Yeah the release pipeline runs the UI tests in a separate stage after every env deployment stage.

**Pull request merged**